app/views/shared/_feedback.html.erb

```diff
@@ -1,15 +1,9 @@
-<% if Feature.active?(:feedback) %>
-  <section class="panel feedback-panel">
-    <h2 class="feedback-panel__title"><%= t('feedback_panel.title') %></h2>
-    <p><%= t('feedback_panel.greeting') %></p>
-    <ul>
-      <li><%= link_to t('feedback_panel.article_feedback_link_text'), new_article_feedback_path(article.object.id), class: 't-feedback-panel__article-feedback-link' %></li>
-      <li><%= link_to t('feedback_panel.technical_feedback_link_text'), new_technical_feedback_path, class: 't-feedback-panel__technical-feedback-link' %></li>
-      <li><%= link_to t('feedback_panel.advice_link_text'), advice_path, class: 't-feedback-panel__advice-link' %></li>
-    </ul>
-  </section>
-<% else %>
-  <section class="inline-feedback">
-    <%= link_to t('inline_feedback.technical_feedback_link'), new_technical_feedback_path, class: 'inline-feedback__link t-feedback-panel__technical-feedback-link' %>
-  </section>
-<% end %>
+<section class="panel feedback-panel">
+  <h2 class="feedback-panel__title"><%= t('feedback_panel.title') %></h2>
+  <p><%= t('feedback_panel.greeting') %></p>
+  <ul>
+    <li><%= link_to t('feedback_panel.article_feedback_link_text'), new_article_feedback_path(article.object.id), class: 't-feedback-panel__article-feedback-link' %></li>
+    <li><%= link_to t('feedback_panel.technical_feedback_link_text'), new_technical_feedback_path, class: 't-feedback-panel__technical-feedback-link' %></li>
+    <li><%= link_to t('feedback_panel.advice_link_text'), advice_path, class: 't-feedback-panel__advice-link' %></li>
+  </ul>
+</section>
```

Comment:
@tomas-stefano I'm going to remove this commit from this branch and create a separate tech debt task to look for and clean up any forgotten feature toggles. I want to check this out a bit more before making the changes.